internal/host/hostcommands/install_cmd.go

```diff
@@ -186,26 +187,28 @@ func (i *installCmd) getFullInstallerCommand(cluster *common.Cluster, host *mode
 
 func (i *installCmd) getProxyArguments(clusterName, baseDNSDomain, httpProxy, httpsProxy, noProxy string) []string {
 	cmd := make([]string, 0)
+	if httpProxy == "" && httpsProxy == "" {
+		return cmd
+	}
 
-	if httpProxy != "" || httpsProxy != "" {
-		if httpProxy != "" {
-			cmd = append(cmd, "--http-proxy", httpProxy)
-		}
-		if httpsProxy != "" {
-			cmd = append(cmd, "--https-proxy", httpsProxy)
-		}
+	if httpProxy != "" {
+		cmd = append(cmd, "--http-proxy", httpProxy)
+	}
+	if httpsProxy != "" {
+		cmd = append(cmd, "--https-proxy", httpsProxy)
+	}
 
+	noProxyUpdated := []string{noProxy}
+	if noProxy != "*" {
```

Comment:
Maybe worth adding a small comment on top of this if to clarify why this statement is here. wdyt?